### DISCUSSION TITLE
Questions related to Qdrant on flexibility and offline support?

### DISCUSSION THREAD

**Discussion by Pratyush2758** [qdrant/qdrant#4942] [Q&A]
Hi,
I wanted to ask few things related to Qdrant Vector Database. 

- As an open source vector database, can Qdrant be self hosted in the local environment? 
- Also if needed in the future, will Qdrant be able to provide the Offline support or not? If yes, the supportive links of docs would be really helpful.

**Comment by azayarni** [comment]
Local environment with Docker. https://qdrant.tech/documentation/quickstart/ 
What is meant by offline support?

**Reply by Pratyush2758** [reply]
Given the confidential nature of our data and our requirement to host the vector database within our AWS environment, we would like to know about the support options available from Qdrant. If we encounter challenges or issues with Qdrant, what kind of support and guidance can Qdrant provide? Are there dedicated support for Qdrant distributed enterprise users on AWS?

**Comment by azayarni** [comment]
Yes, there are support packages available. For details, please reach out via the contact form. https://qdrant.tech/contact-us/

**Reply by Pratyush2758** [reply]
Okay thanks for the response @azayarni. But just wanted to reiterate, Qdrant can be self hosted, right?

**Reply by timvisee** [reply]
> But just wanted to reiterate, Qdrant can be self hosted, right?

Yes.

You may be interested in [Hybrid Cloud](https://qdrant.tech/documentation/hybrid-cloud/), which provides a brilliant hybrid between self hosting while being managed.

Otherwise you can use it through Docker ([details](https://qdrant.tech/documentation/guides/installation/#docker)):

```bash
docker run -p 6333:6333 \
    -v $(pwd)/path/to/data:/qdrant/storage \
    qdrant/qdrant
```
